demos/test-app/src/index.tsx

```diff
@@ -215,7 +215,6 @@ const readCanisterId = (): string => {
 };
 
 const init = async () => {
-  await updateAlternativeOriginsView();
```

Comment:
Moving this function and `await` at the end of the `init` helped fix some e2e tests.